source/server/admin/stats_params.h

```diff
@@ -40,6 +40,12 @@ enum class StatsType {
   All,
 };
 
+enum class HiddenFlag {
+  Include,
```

Comment:
add inline comments explaining what these choices get you.

Reply:
Done